oauth2_http/javatests/com/google/auth/oauth2/ServiceAccountCredentialsTest.java

```diff
@@ -975,6 +976,58 @@ public void idTokenWithAudience_iamFlow_targetAudienceDoesNotMatchAudClaim() thr
         tokenCredential.getIdToken().getJsonWebSignature().getPayload().getAudience());
   }
 
+  @Test
+  public void idTokenWithAudience_oauthEndpoint_non2XXStatusCode() throws IOException {
+    String universeDomain = "test.com";
```

Comment:
is this UD necessary? I'd prefer to only test non-gdu domain with tests dedicated to tpc

Reply:
SG, removing for this one.